client/source/class/cv/io/transport/LongPolling.js

```diff
@@ -208,6 +211,14 @@ qx.Class.define('cv.io.transport.LongPolling', {
     handleError: qx.core.Environment.select("cv.xhr", {
       "qx": function (ev) {
         var req = ev.getTarget();
+        // check for temporary server errors and retry a few times
+        if([408,444,499,502,503,504].indexOf(req.getStatus()) >= 0 && this.retryServerErrorCounter < this.client.backend.maxRetries) {
+          this.info('Temporary connection problem (status: ' + req.getStatus() + ') - retry count: ' + this.retryServerErrorCounter);
+          this.retryServerErrorCounter++;
+          req.serverErrorHandled = true;
+          this.restart();
+          return;
+        }
```

Comment:
The retry-handling feature should also be added to the jquery version of the client. Otherwise the two client releases would behave differently.

Reply:
As I have no way to test as the moment I didn't want to take the risk for such a change at a very critical key component of the CV.
The PR is "just" improving robustness, so I think it's ok to have a slightly different behaviour

Reply:
Ok, I assume that there are not many (or even no) users for the standalone client in the jquery version, I guess we can live with that difference.